<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QGroupBox" name="smileDetailBox" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>460</x>
      <y>130</y>
      <width>321</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title" >
     <string>Edit smile</string>
    </property>
    <widget class="QListWidget" name="textList" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>120</y>
       <width>94</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="textEdit" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>290</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="saveButton" >
     <property name="geometry" >
      <rect>
       <x>50</x>
       <y>360</y>
       <width>75</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text" >
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelButton" >
     <property name="geometry" >
      <rect>
       <x>200</x>
       <y>360</y>
       <width>75</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text" >
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QLabel" name="movieLabel" >
     <property name="geometry" >
      <rect>
       <x>30</x>
       <y>30</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="layoutDirection" >
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap/>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QToolButton" name="saveTextButton" >
     <property name="geometry" >
      <rect>
       <x>110</x>
       <y>290</y>
       <width>27</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset resource="MainWindow.qrc" >
       <normaloff>:/smileManager/images/accept32.png</normaloff>:/smileManager/images/accept32.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="addTextButton" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>90</y>
       <width>27</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset resource="MainWindow.qrc" >
       <normaloff>:/smileManager/images/add32.png</normaloff>:/smileManager/images/add32.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="removeTextButton" >
     <property name="geometry" >
      <rect>
       <x>40</x>
       <y>90</y>
       <width>27</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset resource="MainWindow.qrc" >
       <normaloff>:/smileManager/images/delete32.png</normaloff>:/smileManager/images/delete32.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="changeImageButton" >
     <property name="geometry" >
      <rect>
       <x>80</x>
       <y>40</y>
       <width>27</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset resource="MainWindow.qrc" >
       <normaloff>:/smileManager/images/folder32.png</normaloff>:/smileManager/images/folder32.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="packBox" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title" >
     <string>Pack Info</string>
    </property>
    <widget class="QLineEdit" name="nameEdit" >
     <property name="geometry" >
      <rect>
       <x>60</x>
       <y>20</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="nameLabel" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Name</string>
     </property>
     <property name="textFormat" >
      <enum>Qt::AutoText</enum>
     </property>
     <property name="buddy" >
      <cstring>nameEdit</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="authorLabel" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>50</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Author</string>
     </property>
     <property name="textFormat" >
      <enum>Qt::AutoText</enum>
     </property>
     <property name="buddy" >
      <cstring>authorEdit</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="authorEdit" >
     <property name="geometry" >
      <rect>
       <x>60</x>
       <y>50</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="versionEdit" >
     <property name="geometry" >
      <rect>
       <x>300</x>
       <y>20</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="versionLabel" >
     <property name="geometry" >
      <rect>
       <x>250</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Version</string>
     </property>
     <property name="textFormat" >
      <enum>Qt::AutoText</enum>
     </property>
     <property name="buddy" >
      <cstring>versionEdit</cstring>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="smileListBox" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>130</y>
      <width>401</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title" >
     <string>Smile list</string>
    </property>
    <widget class="QTableWidget" name="table" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>50</y>
       <width>381</width>
       <height>351</height>
      </rect>
     </property>
     <property name="editTriggers" >
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior" >
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="columnCount" >
      <number>3</number>
     </property>
     <column>
      <property name="text" >
       <string>Icon</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Icon filename</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Text</string>
      </property>
     </column>
    </widget>
    <widget class="QToolButton" name="downSmileButton" >
     <property name="geometry" >
      <rect>
       <x>130</x>
       <y>20</y>
       <width>27</width>
       <height>25</height>
      </rect>
     </property>
     <property name="icon" >
      <iconset resource="MainWindow.qrc" >
       <normaloff>:/smileManager/images/down32.png</normaloff>:/smileManager/images/down32.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="removeSmileButton" >
     <property name="geometry" >
      <rect>
       <x>70</x>
       <y>20</y>
       <width>27</width>
       <height>25</height>
      </rect>
     </property>
     <property name="icon" >
      <iconset resource="MainWindow.qrc" >
       <normaloff>:/smileManager/images/delete32.png</normaloff>:/smileManager/images/delete32.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="upSmileButton" >
     <property name="geometry" >
      <rect>
       <x>100</x>
       <y>20</y>
       <width>27</width>
       <height>25</height>
      </rect>
     </property>
     <property name="icon" >
      <iconset resource="MainWindow.qrc" >
       <normaloff>:/smileManager/images/up32.png</normaloff>:/smileManager/images/up32.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="addSmileButton" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>20</y>
       <width>27</width>
       <height>25</height>
      </rect>
     </property>
     <property name="icon" >
      <iconset resource="MainWindow.qrc" >
       <normaloff>:/smileManager/images/add32.png</normaloff>:/smileManager/images/add32.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="editSmileButton" >
     <property name="geometry" >
      <rect>
       <x>40</x>
       <y>20</y>
       <width>27</width>
       <height>25</height>
      </rect>
     </property>
     <property name="icon" >
      <iconset resource="MainWindow.qrc" >
       <normaloff>:/smileManager/images/edit32.png</normaloff>:/smileManager/images/edit32.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>Import</string>
    </property>
    <addaction name="actionImport_From_Kopete" />
    <addaction name="actionImport_From_Pidgin_ZIP" />
    <addaction name="actionImport_From_Pidgin_Folder" />
    <addaction name="actionImport_From_QIP_ZIP" />
    <addaction name="actionImport_From_QIP_Folder" />
   </widget>
   <widget class="QMenu" name="menuExport" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="title" >
     <string>Export</string>
    </property>
    <addaction name="actionExport_To_Pidgin" />
    <addaction name="actionExport_To_Kopete" />
    <addaction name="actionExport_To_Qip" />
    <addaction name="actionExport_To_All" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuExport" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionImport_From_Kopete" >
   <property name="text" >
    <string>Import From Kopete</string>
   </property>
  </action>
  <action name="actionImport_From_Pidgin_ZIP" >
   <property name="text" >
    <string>Import From Pidgin ZIP</string>
   </property>
  </action>
  <action name="actionImport_From_Pidgin_Folder" >
   <property name="text" >
    <string>Import From Pidgin Folder</string>
   </property>
  </action>
  <action name="actionImport_From_QIP_ZIP" >
   <property name="text" >
    <string>Import From QIP ZIP</string>
   </property>
  </action>
  <action name="actionImport_From_QIP_Folder" >
   <property name="text" >
    <string>Import From QIP Folder</string>
   </property>
  </action>
  <action name="actionExport_To_Pidgin" >
   <property name="text" >
    <string>Export To Pidgin</string>
   </property>
  </action>
  <action name="actionExport_To_Kopete" >
   <property name="text" >
    <string>Export To Kopete</string>
   </property>
  </action>
  <action name="actionExport_To_Qip" >
   <property name="text" >
    <string>Export To Qip</string>
   </property>
  </action>
  <action name="actionExport_To_All" >
   <property name="text" >
    <string>Export To All</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="MainWindow.qrc" />
 </resources>
 <connections/>
</ui>
